modules/appeals_api/app/models/appeals_api/notice_of_disagreement.rb

```diff
@@ -259,20 +259,10 @@ def update_status(status:, code: nil, detail: nil, raise_on_error: false)
           }.deep_stringify_keys
         )
 
-        return if auth_headers.blank? # Go no further if we've removed PII
```

Comment:
This converstion from `auth_header` to `form_data` doesn't change the logic here, since both are [cleared at the same time in our PII removal job](https://github.com/department-of-veterans-affairs/vets-api/blob/03d7cf87680fc491d81f5ea4bc0f1f12a5fef073/modules/appeals_api/app/services/appeals_api/remove_pii.rb#L32), but since the segmented APIs store their most important PII in `form_data`, I figured it wouldn't hurt to focus on that here instead of the headers